public/js/actions/subscriptions.js

```diff
@@ -0,0 +1,33 @@
+import $ from 'jquery';
+
+import * as actionTypes from 'constants/action-types';
+
+import * as appActions from './app';
+
+
+export function getUserSubscriptions(jquery=$) {
+  return dispatch => {
+    dispatch({
+      type: actionTypes.LOADING_USER_SUBSCRIPTIONS,
```

Comment:
This is a different convention to the one where we're using a component that calls the action and just returns a spinner. Should we consolidate that (either way)?